ivy/functional/frontends/paddle/manipulation.py

```diff
@@ -173,3 +173,24 @@ def unstack(x, axis=0, name=None):
 
 
 absolute = abs
+
+#crrop
+# Load an image (you'll need to have an image file)
+image_path = "input_image.jpg"  # Replace with your image file path
```

Comment:
Please remove this code. This is not relevant.

Reply:
I have been trying to link the issue and the paddle doc bit i haven't been able to. 
Can you guide me through it, please?

Reply:
Sure, I will remove the codes

Reply:
You have to implement the frontend as explained here https://unify.ai/docs/ivy/overview/contributing/open_tasks.html#frontend-apis